components/clarinet-deployments/src/lib.rs

```diff
@@ -125,14 +125,14 @@ pub fn update_session_with_contracts_executions(
 ) -> BTreeMap<QualifiedContractIdentifier, Result<ExecutionResult, Vec<Diagnostic>>> {
     let boot_contracts_data = BOOT_CONTRACTS_DATA.clone();
 
-    for (_, (boot_contract, ast)) in boot_contracts_data {
-        session
+    let mut results = BTreeMap::new();
```

Comment:
When I originally made this change, I wondered if this has any side effects that I hadn't considered. Can you think of any ways that adding these boot contracts to the contract executions could be problematic @hugocaillard?

Reply:
@MicaiahReid
To avoid unwanted side effects (such has having diagnostics from the boot contracts or even a bad impact on the performances), I split the `update_session_with_contracts_executions` result to separate boot_contracts result from the project contracts